te_chef.py

```diff
@@ -172,22 +177,52 @@ def scrape_content(title, content_url):
         youtube_url = doc.select_one(".video-container iframe")["src"]
         youtube_id = get_youtube_id_from_url(youtube_url)
 
-        # Some of the videos have been removed from the YouTube channel --
-        # skip creating content nodes for them entirely so they don't show up
-        # as non-loadable videos in Kolibri.
         try:
-            ydl.extract_info(youtube_url, download=False)
+            info = ydl.extract_info(youtube_url, download=False)
+            subtitle_languages = info["subtitles"].keys()
+            print ("      ... with subtitles in languages:", subtitle_languages)
         except youtube_dl.DownloadError as e:
-                print("        NOTE: Skipping video download due to error: ", e)
-                return None
+            # Some of the videos have been removed from the YouTube channel --
+            # skip creating content nodes for them entirely so they don't show up
+            # as non-loadable videos in Kolibri.
+            print("        NOTE: Skipping video download due to error: ", e)
+            return None
 
         video_node = nodes.VideoNode(
             **base_node_attributes,
             derive_thumbnail=True,
             files=[files.YouTubeVideoFile(youtube_id=youtube_id)],
         )
 
-        # TODO(davidhu): Get subtitles in other languages
+        # Add subtitles in whichever languages are available.
+        for language in subtitle_languages:
+            if not languages.getlang(language):
+                # It seems like the subtitle language codes that we get as queried
+                # from youtube-dl are not all consistent with the codes of
+                # the languages in le-utils.
+                #
+                # E.g. we may get "zh-Hans" from
+                # youtube-dl but languages.getlang("zh-Hans") returns None
+                # while languages.getlang("zh") returns something.
+                # Another example, we may get "zu" from youtube-dl but
+                # languages.getlang("zu") returns None while
+                # languages.getlang("zul") returns something ("zul" seems
+                # like the ISO-639-3 version of the language code for Zulu).
+                #
+                # Now, though it's possible that we can still find the
+                # corresponding le_utils.languages.getlang for a given
+                # language, we still need to retain the language code in the
+                # form as returned from youtube-dl in order to actually be able
+                # to download that language from YouTube.
+                #
+                # TODO(david): Make a change in Ricecooker so that we can
+                # resolve this issue. As of July 10, 2017, about 13 subtitles
+                # don't get downloaded due to this issue.
```

Comment:
@aronasorman, since you authored the `le_utils.languages` file, if you could take a look at the comment above and let me know if there's anything you'd know of that'd be relevant in order to get this `TODO` comment done, that'd be great, thanks :)